src/AbstractSQLCompiler.ts

```diff
@@ -857,9 +857,12 @@ ${compileRule(definitionAbstractSql as AbstractSqlQuery, engine, true).replace(
 		const createSqlElements: string[] = [];
 		const createIndexes: string[] = [];
 
+		const computedFields: Array<AbstractSqlField['computed']> = [];
```

Comment:
Could you do something like this to avoid the cast later on?
```typescript
const computedFields: Array<['Alias', AbstractSqlQuery, string]> = [];
```